cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceStateMachineTest.java

```diff
@@ -58,6 +62,7 @@
     private static final String RESOURCE_NAME = "my-rebalance";
     private static final String CLUSTER_NAMESPACE = "cruise-control-namespace";
     private static final String CLUSTER_NAME = "kafka-cruise-control-test-cluster";
+    ConfigMapOperator mockCmOps;
```

Comment:
Why is this a being stored as a field in the class? The other mocks are method instances?

Reply:
Have you addressed this @ShubhamRwt?